browser/templates/browser/investigator_edit.html

```diff
@@ -0,0 +1,89 @@
+{% extends "browser/change_base.html" %}
+{% load addcss %}
+
+{% block title %}Edit Investigator: {{ person.name }} (Person){% endblock %}
+{% block main %}
+<div class="alert alert-mbl">
+    Person
+    <div>
+        <span class="h1">{{ person.name }}</span>
+    </div>
+    {% if person.authority %}
+    <div class="text-warning">
+        <span class="glyphicon glyphicon-barcode"></span>
+        {{ person.authority.conceptpower_uri|urlize }}
+    </div>
+    {% endif %}
+    {% if person.validated %}
+    <div>
+        <span class="glyphicon glyphicon-ok" style="color:green;"></span>&nbsp;
+        Validated by {{ person.validated_by }} on {{ person.validated_on }}.
+    </div>
+    {% endif %}
+</div>
+
+<div class="container">
+    <p class="text-info">
+            The below form will edit this <strong>Investigator</strong> record for <strong>{{ person.name }}</strong>.
+        </p>
+
+    <form action="." method="post" id="InvestigatorForm">
+    {% csrf_token %}
+    {% block form_fields %}
+    {% with form.subject as field %}
+    <div class="row">
+        <div class="col-xs-12">
+        <div class="form-group">
+            {% for error in field.errors %}
+            <div class="alert alert-warning">{{ error }}</div>
+            {% endfor %}
+            <label>{{ field.label }}</label>
+            {{ field|addcss:"form-control input-sm" }}
+            <p class="text-muted">{{ field.help_text }}</p>
+        </div>
+        </div>
+    </div>
+    {% endwith %}
+
+    {% with form.role as field %}
+    <div class="row">
+        <div class="col-xs-8">
+        <div class="form-group">
+            {% for error in field.errors %}
+            <div class="alert alert-warning">{{ error }}</div>
+            {% endfor %}
+            <label>{{ field.label }}</label>
+            {{ field|addcss:"form-control input-sm" }}
+            <p class="text-muted">{{ field.help_text }}</p>
+        </div>
+        </div>
+    {% endwith %}
+
+    {% with form.year as field %}
+        <div class="col-xs-4">
+        <div class="form-group">
+            {% for error in field.errors %}
+            <div class="alert alert-warning">{{ error }}</div>
+            {% endfor %}
+            <label>{{ field.label }}</label>
+            {{ field|addcss:"form-control input-sm" }}
+            <p class="text-muted">{{ field.help_text }}</p>
+        </div>
+        </div>
+    </div>
+    {% endwith %}
+    {% endblock %}
+    </form>
+
+    <div class="row">
+    <div class="text-center">
+        <form action="."  method="post">
+            {% csrf_token %}
+            <input class="btn btn-success" type="submit" form="InvestigatorForm" name="UpdateButton_Clicked" value="Update">
+            <input class="btn btn-danger" type="submit" value="Cancel" name="CancelButton_Clicked">
+        </form>
+    </div>
+    </div>
+
+</div>
+{% endblock %}
```

Comment:
you should be able to use the same template for create and upate since the only difference is that there already exist an instance.

Reply:
Changed code so that only one template is used to create and update investigator